test/functional-api/communications/community-updates/updates.it-spec.ts

```diff
@@ -0,0 +1,233 @@
+import {
+  getSpaceDataCodegen,
+  deleteSpaceCodegen,
+} from '@test/functional-api/journey/space/space.request.params';
+import { deleteOrganizationCodegen } from '@test/functional-api/organization/organization.request.params';
+import { TestUser } from '@test/utils/token.helper';
+import { entitiesId } from '@test/functional-api/zcommunications/communications-helper';
+import { uniqueId } from '@test/utils/mutations/create-mutation';
+import { changePreferenceSpaceCodegen } from '@test/utils/mutations/preferences-mutation';
+import { users } from '@test/utils/queries/users-data';
+import { assignCommunityRoleToUserCodegen } from '@test/functional-api/integration/community/community.request.params';
+import { createOrgAndSpaceCodegen } from '@test/utils/data-setup/entities';
+import { CommunityRole, SpacePreferenceType } from '@alkemio/client-lib';
+import {
+  removeMessageOnRoomCodegen,
+  sendMessageToRoomCodegen,
+} from '../communication.params';
+const organizationName = 'upd-org-name' + uniqueId;
+const hostNameId = 'upd-org-nameid' + uniqueId;
+const spaceName = 'upd-eco-name' + uniqueId;
+const spaceNameId = 'upd-eco-nameid' + uniqueId;
+
+beforeAll(async () => {
+  await createOrgAndSpaceCodegen(
+    organizationName,
+    hostNameId,
+    spaceName,
+    spaceNameId
+  );
+});
+
+afterAll(async () => {
+  await deleteSpaceCodegen(entitiesId.spaceId);
+  await deleteOrganizationCodegen(entitiesId.organizationId);
+});
+
+describe('Communities', () => {
+  describe('Community updates - read access', () => {
+    beforeAll(async () => {
+      await changePreferenceSpaceCodegen(
+        entitiesId.spaceId,
+        SpacePreferenceType.AuthorizationAnonymousReadAccess,
+        'false'
+      );
+
+      await assignCommunityRoleToUserCodegen(
+        users.spaceMemberId,
+        entitiesId.spaceCommunityId,
+        CommunityRole.Member
+      );
+
+      const res = await sendMessageToRoomCodegen(
+        entitiesId.spaceUpdatesId,
+        'test'
+      );
+      entitiesId.messageId = res?.data?.sendMessageToRoom.id;
+    });
+
+    afterAll(async () => {
+      await removeMessageOnRoomCodegen(
+        entitiesId.spaceUpdatesId,
+        entitiesId.messageId
+      );
+    });
+    test('community updates - PRIVATE space - read access - sender / reader (member) / reader (not member)', async () => {
+      // Act
+      const spaceDataSender = await getSpaceDataCodegen(
+        entitiesId.spaceId,
+        TestUser.GLOBAL_ADMIN
+      );
+      const getMessageSender =
+        spaceDataSender?.data?.space?.community?.communication?.updates
+          .messages ?? '';
```

Comment:
Rename + it shouldn't be defaulted to ''. This is an array of message object. It should be [].